tools/build.py

```diff
@@ -9,38 +11,48 @@
 PREFIX = "cba_"
 ##########################
 
-def tryHemttBuild(projectpath):
-    hemttExe = os.path.join(projectpath, "hemtt.exe")
-    if os.path.isfile(hemttExe):
+def try_hemtt_build(projectpath):
+    """Try building with hemtt"""
+    hemtt = os.path.join(projectpath, "hemtt.exe")
+    if os.path.isfile(hemtt):
         os.chdir(projectpath)
-        ret = subprocess.call([hemttExe, "pack"], stderr=subprocess.STDOUT)
-        print("Using hemtt: {}".format(ret));
+        ret = subprocess.call([hemtt, "pack"], stderr=subprocess.STDOUT)
+        print("Using hemtt: {}".format(ret))
         return True
-    else:
-        print("hemtt not installed");
+
+    print("hemtt not installed")
     return False
 
 def mod_time(path):
+    """Return time of last modification of a file or directory
+
+    If the path is a directory, return the time of modification of
+    the most recently modified file in the directory.
+    """
     if not os.path.isdir(path):
         return os.path.getmtime(path)
     maxi = os.path.getmtime(path)
-    for p in os.listdir(path):
-        maxi = max(mod_time(os.path.join(path, p)), maxi)
+    for name in os.listdir(path):
+        maxi = max(mod_time(os.path.join(path, name)), maxi)
     return maxi
 
 
 def check_for_changes(addonspath, module):
-    if not os.path.exists(os.path.join(addonspath, "{}{}.pbo".format(PREFIX,module))):
+    """Return True if the sources of an addon have been changed since it was last built"""
+    file = os.path.join(addonspath, "{}{}.pbo".format(PREFIX, module))
+    if not os.path.exists(file):
         return True
-    return mod_time(os.path.join(addonspath, module)) > mod_time(os.path.join(addonspath, "{}{}.pbo".format(PREFIX,module)))
+    return mod_time(os.path.join(addonspath, module)) > mod_time(file)
 
 def check_for_obsolete_pbos(addonspath, file):
+    """Return True if the source directory for addon no longer exists"""
     module = file[len(PREFIX):-4]
     if not os.path.exists(os.path.join(addonspath, module)):
         return True
     return False
 
 def main():
+    """main"""
```

Comment:
As much as I like standards, Python linters go too far even for my taste. This line is just super redundant.

Reply:
Yeah, I'll add a pyling ignore for this one